<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page"/>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_export"/>
    <addaction name="action_import"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="action_showchart"/>
    <addaction name="separator"/>
    <addaction name="action_history"/>
    <addaction name="separator"/>
    <addaction name="action_analysis"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action_export">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/export.png</normaloff>:/resource/export.png</iconset>
   </property>
   <property name="text">
    <string>导出数据</string>
   </property>
  </action>
  <action name="action_import">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/import.png</normaloff>:/resource/import.png</iconset>
   </property>
   <property name="text">
    <string>导入数据</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/exit.png</normaloff>:/resource/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;F)</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/disconnect.png</normaloff>:/resource/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/connect.png</normaloff>:/resource/connect.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/readdata.png</normaloff>:/resource/readdata.png</iconset>
   </property>
   <property name="text">
    <string>读取数据</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/about.png</normaloff>:/resource/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/resource/qt.png</normaloff>:/resource/qt.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="action_showchart">
   <property name="text">
    <string>实时数据显示</string>
   </property>
  </action>
  <action name="action_history">
   <property name="text">
    <string>历史数据查看</string>
   </property>
  </action>
  <action name="action_analysis">
   <property name="text">
    <string>分析数据
</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
